demos/demo-react-ts/src/components/App.tsx

```diff
@@ -174,11 +174,11 @@ function App() {
 
   cti.broadcastChannel.onmessage = ({
     data,
-  }: MessageEvent<{ type: string }>) => {
+  }: MessageEvent<{ type: string; payload?: any }>) => {
     // Send SDK message to HubSpot
     if (iframeLocation === "window") {
       const eventHandler = broadcastEventHandlers[data.type];
-      if (eventHandler) {
+      if (eventHandler && cti[eventHandler] && data.payload) {
```

Comment:
can there be an event which doesnt take payload?

Reply:
Yeah like `this.broadcastMessage({ type: thirdPartyToHostEvents.INITIALIZED });`

Reply:
in that case, will that event handler get skipped out in above line.

Reply:
Good point. Will follow up with this in another PR!